src/AasxServerStandardBib/Database.cs

```diff
@@ -0,0 +1,258 @@
+using MongoDB.Bson.Serialization;
+using MongoDB.Driver;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AasxServerStandardBib.Exceptions;
+using MongoDB.Bson;
+using MongoDB.Driver.Linq;
+using System.Collections;
+using AasCore.Aas3_0;
+using AdminShellNS;
+using MongoDB.Driver.GridFS;
+using static AasxServerStandardBib.TimeSeriesPlotting.PlotArguments;
+using System.IO;
+
+
+//Author: Jonas Graubner
+//contact: jogithub@graubner-bayern.de
+namespace AasxServerStandardBib
+{
+    public interface IDatabase
+    {
+        void Initialize(String connectionString);
+
+        #region AssetAdministrationShell
+        public void WriteDBAssetAdministrationShell(IAssetAdministrationShell shell);
+        public IQueryable<AssetAdministrationShell> GetLINQAssetAdministrationShell();
+        public void UpdateDBAssetAdministrationShellById(IAssetAdministrationShell body, string aasIdentifier);
+        public bool DeleteDBAssetAdministrationShellById(IAssetAdministrationShell shell);
+        #endregion
+
+        #region Submodel
+        public void WriteDBSubmodel(ISubmodel submodel);
+        public IQueryable<Submodel> GetLINQSubmodel();
+        public void UpdateDBSubmodelById(string submodelIdentifier, ISubmodel newSubmodel);
+        public void DeleteDBSubmodelById(string submodelIdentifier);
+        #endregion
+
+        #region ConceptDescription
+        public void WriteDBConceptDescription(IConceptDescription conceptDescription);
+        public IQueryable<ConceptDescription> GetLINQConceptDescription();
+        public void UpdateDBConceptDescriptionById(IConceptDescription newConceptDescription, string cdIdentifier);
+        public void DeleteDBConceptDescriptionById(string conceptDescription);
+        #endregion
+
+        #region Filestream
+        public void WriteFile(Stream stream, string filename);
+        public Stream ReadFile(string filename);
+        public void DeleteFile(string filename);
+        #endregion
+
+        public void importAASCoreEnvironment(IEnvironment environment);
+        public void importAdminShellPackageEnv(AdminShellPackageEnv adminShellPackageEnv);
+    }
+
+
+    public class MongoDatabase : IDatabase
+    {
+        private MongoClient _client;
+        private IMongoDatabase _database;
+        private GridFSBucket _bucket;
+
+        public void Initialize(String connectionString)
+        {
+            //_client = new MongoClient("mongodb://AAS:SefuSWE63811!@192.168.0.22:27017/?authSource=AAS");
+            _client = new MongoClient(connectionString);
+            try
+            {
+                _client.StartSession();
+            }
+            catch (System.TimeoutException ex)
+            {
+                System.Console.WriteLine(ex.Message);
+                System.Environment.Exit(1);
+            }
+
+            _database = _client.GetDatabase("AAS");
```

Comment:
If the values like `AAS`, `AasCore`, and the bucketName `aasxFiles` are constant, it's best practice to use class constants. This helps avoid "Magic Numbers" and improves code readability and maintainability.

For more information, see [Code Smell: Magic Numbers](https://luzkan.github.io/smells/magic-number).